### PR TITLE
Prepare 2.10.1 release

### PR DESCRIPTION
**Pull request by gjsjohnmurray**

### CODE REVIEW

**Comment by isc-bsaviano** [comment]
@gjsjohnmurray Can this wait until I remove the query parameters I deprecated in #1165?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+## [2.10.1] 10-Aug-2023
+- Enhancements
+  - Only add WebSocket Terminal button to Server Manager 3.4.2+ tree if server is compatible (#1204)
+  - Add `Copy Invocation` CodeLens above query definition in class (#1198)
+  - Remove previously deprecated `objectscript.serverSideEditing` setting (#1195)
```
Comment (isc-bsaviano) [inline CHANGELOG.md:7]:
```suggestion
  - Remove reference to non-existent `objectscript.serverSideEditing` setting (#1195)
```

**Comment by gjsjohnmurray** [comment]
> @gjsjohnmurray Can this wait until I remove the query parameters I deprecated in #1165?

OK. Are you ready to do that now, or do you want to wait longer?

**Comment by isc-bsaviano** [comment]
I should be able to get that done by the end of the day. I'm on vacation Friday through Tuesday.

**Comment by gjsjohnmurray** [comment]
On second thoughts, perhaps we should leave the deprecation-handling in place a bit longer. It's only 3 weeks since it shipped, which isn't long for a user to have opened all of their workspaces that might use the params. Are you concerned about the code remaining, other than that removing it will simplify?

**Comment by isc-bsaviano** [comment]
@gjsjohnmurray It can wait

**Pull request merged**